src/main/resources/component-biz.conf

```diff
@@ -1357,8 +1358,7 @@ security {
     subScopes = ${?security.subScopes} ["ui", "api", "vfs", "empty"]
 
     # Defines all tenant permissions (aka features). This also can be redefined by the application.
-    # Init to an empty array, in case it is not redefined somewhere else
-    tenantPermissions = ${?security.tenantPermissions} []
+    tenantPermissions = ${?security.tenantPermissions} ["permission-bypass-process-log-limits"]
```

Comment:
shouldn't we name this tenant-permission-XX or the like to not confuse this with a user permisson? or a feature- or whatever we use elsewhere...

Reply:
Do we want to have an intermediate permission that gives you the actual permission that is being checked or should the existing permission simply be called "tenant-permission-bypass-process-log-limits"?

We do not have an existing naming scheme for tenant permissions yet in the sirius framework. S2 and OX have completely different naming schemes.